homeassistant/components/simplisafe/__init__.py

```diff
@@ -137,8 +142,12 @@
             lambda value: value.total_seconds(),
             vol.Range(min=30, max=480),
         ),
-        vol.Optional(ATTR_ALARM_VOLUME): vol.All(vol.Coerce(int), vol.In(VOLUMES)),
-        vol.Optional(ATTR_CHIME_VOLUME): vol.All(vol.Coerce(int), vol.In(VOLUMES)),
+        vol.Optional(ATTR_ALARM_VOLUME): vol.All(
+            cv.string, lambda value: VOLUME_MAP[value]
```

Comment:
Not every string is valid, so verify it’s a valid key first, then get the value
```suggestion
            vol.In(VOLUME_MAP), VOLUME_MAP.get
```